src/awkward/operations/ak_from_rdataframe.py

```diff
@@ -3,13 +3,24 @@
 import awkward as ak
 
 
-def from_rdataframe(rdf, columns, offsets_type="int64_t", keep_order=False):
+def from_rdataframe(
+    rdf,
+    columns,
+    highlevel=True,
+    behavior=None,
+    offsets_type="int64_t",
+    keep_order=False,
+):
```

Comment:
I think we need to add these at the end of the argument list, so that we don't break existing positional uses. We also want to make `ak.operations.*` functions use keyword-only arguments where it makes sense, and that currently includes `highlevel` and `behavior`, e.g.:
```suggestion
    offsets_type="int64_t",
    keep_order=False,
    *,
    highlevel=True,
    behavior=None,
):
```

At some point, I think we'd want `offsets_type` and `keep_order` to be keyword-only too, but that will be a breaking change that we might need to do by a deprecation cycle.